core/src/androidTest/java/com/facebook/testing/screenshot/internal/AlbumImplTest.java

```diff
@@ -41,6 +41,7 @@
   private String mBarFile;
   private HostFileSender mHostFileSender;
   private ScreenshotDirectories mScreenshotDirectories;
+  private ScreenshotNameCalculator mNameCalculator;
```

Comment:
I think the core java library shouldn't have a knowledge of this, instead the gradle/python wrappers should either rename the files, or put screenshots from each configuration into a different directory

Reply:
Yep, I will open a new PR with changes in python gradle plugin which will be in charge to save screenshots from each configuration in different directories 👍